tests/org.eclipse.swt.tests.win32/ManualTests/org/eclipse/swt/widgets/PatternWin32ManualTest.java

```diff
@@ -0,0 +1,51 @@
+package org.eclipse.swt.widgets;
+
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.graphics.Pattern;
+import org.eclipse.swt.internal.DPIUtil;
+
+public class PatternWin32ManualTest {
+	private static Display display = Display.getDefault();
+
+	public static void main (String [] args) {
```

Comment:
Extract a private method and call it twice with the proper parameters to improve readability. Write the JavaDoc for the extracted method and briefly document its parameters.

Reply:
Done